### PR TITLE
OCI: Add loki unit test

### PR DESCRIPTION
**Pull request by sergiodj**
This PR adds the unit tests for the loki ROCK image.  The first test
is a very simple one; it just makes sure that we can start the loki
service without any configuration changes.  The second test is a bit
more involved: it sets up a promtail service that will be responsible
for monitoring the logs under `/var/log/*.log` and sending them to a
loki service that's running in another container.

These tests should be enough to do cover the basic set of features
from loki.

### CODE REVIEW

**Comment by paride** [comment]
Hi @sergiodj, is `ubuntu/loki` supposed to be generally available already? I fail to pull it at the moment.

File: oci-unit-tests/loki_test.sh
Comment (athos-ribeiro) [inline oci-unit-tests/loki_test.sh:141]: Could we use $LOKI_PORT here (and below)?

**Review by athos-ribeiro — COMMENTED** [review]
Nice work, Sergio :)

I left a few inline comments.

The tests look solid and seem to cover basic features, as we do for the other OCIs.

The tests pass with an image built with the Dockerfile from https://launchpad.net/~canonical-server/ubuntu-docker-images/+git/loki (Cc @paride)

```
test_loki_up
test_loki_and_promtail
test_manifest_exists

Ran 3 tests.

OK
```

File: oci-unit-tests/loki_test.sh
Comment (sergiodj) [reply oci-unit-tests/loki_test.sh:64]: Indeed.  Done.

File: oci-unit-tests/loki_test.sh
Comment (sergiodj) [reply oci-unit-tests/loki_test.sh:141]: Good catch, thanks.  Fixed.

**Comment by sergiodj** [comment]
Thanks for the review, @athos-ribeiro!  I force-pushed a new version of the branch which should address everything you raised.

@paride, as @athos-ribeiro mentioned, the image hasn't been published on Dockerhub/AWS yet.  You will need to build it locally in order to run the tests.

**Review by athos-ribeiro — APPROVED** [review]

**Comment by athos-ribeiro** [comment]
Thanks for addressing the comments, Sergio :)

**Comment by sergiodj** [comment]
Thanks, Athos.

I will wait for @paride's ACK before I merge the branch.

**Review by paride — APPROVED** [review]
Normally I like to "test the test" on Jenkins before merging, but as the image is not available yet in the registries we can't do it this time. In any case this LGTM, I'm going ahead and merging.

@sergiodj @athos-ribeiro once the image is published ping me and I'll add the Jenkins jobs.

Thanks!

**Pull request merged**
